docs/articles/mirroring.md

```diff
@@ -12,11 +12,23 @@ For mirroring an upstream registry, two common use cases are a fully mirrored or
 
 As with git, wherein every clone is a full repository, you can configure your local zot instance to be a fully mirrored OCI registry. For this mode, configure zot for synchronization by periodic polling, not on-demand. Zot copies and caches a full copy of every image on the upstream registry, updating the cache whenever polling discovers a change in content or image version at the upstream registry. 
 
- For a pull through cache mirrored registry, configure zot for on-demand synchronization. When an image is first requested from the local zot registry, the image is downloaded from the upstream registry and cached in local storage. Subsequent requests for the same image are served from zot's cache. Images that have not been requested are not downloaded. If a polling interval is also configured, zot periodically polls the upstream registry for changes, updating any cached images if changes are detected. 
+For a pull through cache mirrored registry, configure zot for on-demand synchronization. When an image is first requested from the local zot registry, the image is downloaded from the upstream registry and cached in local storage. Subsequent requests for the same image are served from zot's cache. Images that have not been requested are not downloaded. If a polling interval is also configured, zot periodically polls the upstream registry for changes, updating any cached images if changes are detected. 
 
 > :pencil2: 
 > Because Docker Hub rate-limits pulls and does not support catalog listing, do not use polled mirroring with Docker Hub. Use only on-demand mirroring with Docker Hub.
 
+## Migrating or updating a registry using mirroring
+
+Mirroring zot using the `sync` feature allows you to easily migrate a registry. In situations such as the following, zot mirroring provides an easy solution.
+
+- Migrating an existing zot or non-zot registry to a new location. 
+  
+    Provided that the source registry is OCI-compliant for image pulls, you can mirror the registry to a new zot registry, delete the old registry, and reroute network traffic to the new registry.
+
+- Updating (or downgrading) a zot registry. 
+  
+    To minimize downtime during an update, or to avoid any incompatibilities between zot releases that would preclude an in place update, you can bring up a new zot registry with the desired release and then migrate from the existing registry.
+
```

Comment:
for both these options sync config (sync everything) would look like this:

```
				{
					"urls": [
						"https://registry:5000"
					],
					"pollInterval": "12h",
					"onDemand": true,
					"content": [
						{
							"prefix": "**"
						}
					]
				},
```

onDemand can also be false, it's just mandatory to have pollInterval and content with prefix '**' to sync everything.